www/pfSense-pkg-Lightsquid/files/usr/local/pkg/lightsquid.inc

```diff
@@ -328,12 +335,16 @@ ssl.honor-cipher-order	= "enable"
 ssl.cipher-list		= "AES128+EECDH:AES256+EECDH:AES128+EDH:AES256+EDH:AES128-SHA:AES256-SHA:!aNULL:!eNULL:!DSS"
 EOF;
 
-		// Configure IPv4 and IPv6 sockets
+		/*
+		 * IPv4 HTTPS bind must NOT be a socket; otherwise the retarded
+		 * lighttpd thing yet again binds to 80/HTTP no matter what.
+		 * You enable SSL-only IPv4 by not configuring bind for IPv4 at all.
+		 * IPv6 SSL must be configured as a socket
+		 */
 		$lighttpd_ls_conf .= <<<EOF
 
```

Comment:
The server.bind thing yet again created an unwanted HTTP/80 when IPv4/HTTPS it was configured as a socket (So, I moved the port to global scope here and stopped doing a bind there altogether). We want this either HTTP only or HTTPS only on wildcard IPv4 and IPv6.

To summarize this: 
- I've spent ~8 hours on this.
- There are Debian/Ubuntu bugs about this madness and inability of the package maintainers to produce OOB working configuration.
- There are questions on StackOverflow and none of them answers this kind of questions, such as "how to stop repeating that SSL block for each socket, or how to skip HTTP to stop clashing with already existing webserver on the same machine.)

Guys, dunno, but configuring a basic webserver configuration is absolutely NOT supposed to be such incredible PITA. 
- The code should respect the configuration. When I create two sockets specifically bound to a particular port and protocol, last thing I expect is that the webserver will "smartly" create an unconfigured HTTP/80 bind and commit a suicide because 80 is already in use
- Configuring an IPv6 wildcard bind is a royal pain (again, see the Debian/Ubuntu/etc. bugs)
- If that was not enough, things get gradually worse when ```net.ipv6.bindv6only = 1``` comes into play. Again, see Debian and other bugs where people complain about lighttpd listening on IPv6 *only* all of a sudden.
- And the wiki article is completely useless joke which lacks any working examples of common configurations and only confuses things further with version-specific stuff and the  ```net.ipv6.bindv6only = 1``` thing.